versions.csv

```diff
@@ -1,10 +1,10 @@
 AssemblyPath, Version
 iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.36.7
 iothub\service\src\Microsoft.Azure.Devices.csproj, 1.31.4
-shared\src\Microsoft.Azure.Devices.Shared.csproj, 1.27.2
-provisioning\device\src\Microsoft.Azure.Devices.Provisioning.Client.csproj, 1.16.4
-provisioning\transport\amqp\src\Microsoft.Azure.Devices.Provisioning.Transport.Amqp.csproj, 1.13.7
-provisioning\transport\http\src\Microsoft.Azure.Devices.Provisioning.Transport.Http.csproj, 1.12.4
+shared\src\Microsoft.Azure.Devices.Shared.csproj, 1.31.0
```

Comment:
I know this is adding an API and therefore should be a minor version upgrade, but I think there might be some value in retaining this as a patch version upgrade. LTS patches are supposed to be security/critical bug fixes and thus customers wouldn't generally be expecting new APIs to be added.
For this Newtonsoft fix, a case could be made that even though a public API was added, we don't expect users to call that API. (Usage-wise, it is an `internal` API but has been declared as `public` since it will be referenced in other packages).
Bumping the minor version might confuse some users into expecting (or being wary of) additional new APIs.

This is just a thought I had and I'm curious to know what others think.

Reply:
That sounds reasonable! I have no problem with that either.

Reply:
Yes, patch version only for LTS.